proposals/dao/fip_97.ts

```diff
@@ -0,0 +1,60 @@
+import hre, { ethers, artifacts } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+
+/*
+
+OA Proposal 97
+
+Description:
+
+Steps:
+1. Seed Turbo Fuse pool with 10 million Fei
+
+*/
+
+const fipNumber = '97'; // Change me!
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  // 1. Deploy compoound PCV deposit
+  const erc20CompoundPCVDepositFactory = await ethers.getContractFactory('ERC20CompoundPCVDeposit');
+  const turboFusePCVDeposit = await erc20CompoundPCVDepositFactory.deploy(addresses.core, addresses.rariTurboFusePool);
+  await turboFusePCVDeposit.deployTransaction.wait();
+
+  console.log('Turbo PCV Deposit deployed to:  ', turboFusePCVDeposit.address);
+
+  return {
+    turboFusePCVDeposit
+  };
+};
+
+// Do any setup necessary for running the test.
+// This could include setting up Hardhat to impersonate accounts,
+// ensuring contracts have a specific state, etc.
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  console.log(`No actions to complete in setup for fip${fipNumber}`);
```

Comment:
You'll need to impersonate and whitelist the PCV deposit to deposit to the turbo pool

Reply:
Done. Note to self - whitelist this PCV deposit on the TurboAdmin once it has been deployed